lib/src/messages/enums/pause_collection_behavior.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:json_annotation/json_annotation.dart';
+
+enum PauseCollectionBehavior {
+  @JsonValue('keep_as_draft')
+  keepAsDraft,
+  @JsonValue('mark_uncollectible')
+  markUncollectible,
+  @JsonValue('void')
+  voided,
```

Comment:
```suggestion
  void_,
```
To match the name of this value from Stripe, WDYT?